.github/workflows/migration-test.yml

```
@@ -6,7 +6,7 @@ on:
 jobs:
   migration-test:
     runs-on: ubuntu-latest
-    
+ 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)